src/MainDashBoard/components/sideSheet/krCheckIn/KrCheckInInputs.tsx

```diff
@@ -168,24 +204,44 @@ export const KR수정하기 = ({ onCancel }: IKrCheckInProps) => {
   };
 
   useEffect(() => {
-    target && logContent ? setIsActiveBtn(true) : setIsActiveBtn(false);
-  }, [target, logContent]);
+    targetValue && logContent ? setIsActiveBtn(true) : setIsActiveBtn(false);
+  }, [targetValue, logContent]);
+
+  //서버 통신 함수
+  const submitCheckIn = async () => {
+    const data = {
+      keyResultId: keyResultId,
+      target: Number(targetValue),
+      logContent: logContent,
```

Comment:
```suggestion
      keyResultId,
      logContent,
      target: Number(targetValue),
```
객체에서 이름이랑 값이랑 같은 건 이런 식으로 한 번만 선언 해줄 수 있습니다

가독성 더 좋아지니까 이렇게 하는 것도 한 번 고려해 보세요